roadmap/roadmap.json

```diff
@@ -0,0 +1,4330 @@
+[
```

Comment:
Can you please check if you intend to change this? This seems like an ESLint so it looks fine but if you didn't intend to edit it maybe don't commit it?